site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -65,92 +73,92 @@ sudo service fusionauth-search start
 
-We will be using this Username, `terry@example.com` in our test code. When finished, select **Save**.
+We will be using this Username, `terry@example.com` in our test code. When finished, select "**Save**".
 
 ## Register the User to our Application
 
-At this point, we will go ahead and register our test user with the application "**pythonapp**" we created for this project. This is an optional step to the process, but demonstrates how to grant a user access and user roles to applications we create in FusionAuth.
+At this point, we will go ahead and register our test user with the application *pythonapp* we created for this project. This is an optional step to the process, but demonstrates how to grant a user access and user roles to applications we create in FusionAuth.
 
-In the User's menu, beside our test user, select "**Manage**" .
+In the User's menu, beside our test user, select the "**Manage**" icon.
 
 {% include _image.liquid src="/assets/img/blogs/action-manage-button.png" alt="Manage user" class="img-fluid mb-4 mt-2 mx-auto d-block" figure=false %}
 
-On the following **Manage User** page, you'll see a button that says, "**Add registration**."
+On the following Manage User page, you'll see a button on the user profile that says, "**Add registration**".
 
-{% include _image.liquid src="/assets/img/blogs/manage-user-add-registration-screen.png" alt="Manage User Add Registration Screen" class="img-fluid full mb-4" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/2021-manage-user.png" alt="Manage User Add Registration Screen" class="img-fluid full mb-4" figure=false %}
 
 This will take you to a page that looks something like this:
 
-{% include _image.liquid src="/assets/img/blogs/add-user-registration-screen.png" alt="Add User Registration Screen" class="img-fluid full mb-4" figure=false %}
+{% include _image.liquid src="/assets/img/blogs/2021-add-user-registration.png" alt="Add User Registration Screen" class="img-fluid full mb-4" figure=false %}
 
-From here, we will want to select our test application, "**pythonapp**". We will match the username with the same username as our sample user's email address. If our application had user roles defined, we could assign them to this user. We can also define the user's Timezone and set Language preferences.
+From here, we will want to select our test application, *pythonapp*. We will match the username with the same username as our sample user's email address. If our application had user roles defined, we could assign them to this user. We can also define the user's Timezone and set Language preferences.
 
-Once you're done, hit **Save**. Now that we are finished setting up our user, it's time to finish configuring the rest of our testing environment.
+Once you're done, hit "**Save**". Now that we are finished setting up our user, it's time to finish configuring the rest of our testing environment.
 
 ## Installing Python PIP Package Manager
 
```

Comment:
I know this is a change from the previous tutorial author, but can we remove the prompt? https://fusionauth.io/blog/2020/03/10/securely-implement-oauth-in-react/ doesn't show the prompt (when you call `npx`, for example).

If you want to call out the prompt because of the (venv), feel free to display it one time.

Reply:
fixed in latest commit